packages/compass/src/app/application.tsx

```diff
@@ -177,40 +114,43 @@ const Application = View.extend({
     webvitals.getLCP(trackPerfEvent);
     webvitals.getFID(trackPerfEvent);
     webvitals.getCLS(trackPerfEvent);
-  },
+  }
+
   /**
    * Enable all debug output for the development mode.
    */
-  preRender: function () {
+  private enableDevelopmentDebug() {
     if (process.env.NODE_ENV === 'development') {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
       require('debug').enable('mon*,had*');
     }
-  },
+  }
+
   /**
    * Pre-load into the require cache a bunch of expensive modules while the
    * user is choosing which connection, so when the user clicks on Connect,
    * Compass can connect to the MongoDB instance faster.
    */
-  postRender: function () {
+  private preloadSlowModules() {
```

Comment:
I think while you're at it, you can just drop this method completely

Reply:
Is it ok to do that?

Reply:
You can probably time it yourself to make sure, but I'm pretty sure this stopped having any meaningful effect when we switched to webpack bundling everything all those years ago. There's still runtime price to pay for requiring something for the first time even inside webpack bundle, but the code is in the bundle, so it's already parsed and loaded, and I think that's the part that takes the most time